brainzutils/metrics.py

```diff
@@ -66,7 +66,7 @@ def set(metric_name: str, tags: Dict[str, str] = None, timestamp: int = None, **
         else:
             fields_list.append("%s=%s" % (k, str(v)))
 
-    fields = " ".join(fields_list)
+    fields = ",".join(fields_list)
```

Comment:
This needs a full test suite !
For example, would this work with the greek question mark?

Reply:
that being said, the tests did actually fail, so they're doing their job!